rtc/getting-started.md

```diff
@@ -12,27 +12,26 @@ keywords: rtc
 
 * A {{site.cloudname}} API key from [{{site.accountpage}}]({{site.accountsignup}}).
 * A JSON Web Token (JWT) key generated at [{{site.accountpageurl}}]({{site.accountpageurl}}). Temporarily store the generated keys for use in the following procedure.
+* A [secure context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) for your web server. RTC makes use of [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) which is only available in secure contexts.
 
 ## Basic RTC setup
 
-The following example shows how to get the RTC plugin up and running using hardcoded values. For a production setup, the document ID and secret would be retrieved from your server. This example also assumes that a JWT provider endpoint exists at '/jwt'. For information on setting up a JWT endpoint, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
+The following example assumes you are familiar with integrating TinyMCE; if you are new TinyMCE user please follow the [basic setup guide]({{site.baseurl}}/general-configuration-guide/basic-setup/) first.
+
+This example shows how to get the RTC plugin up and running using hardcoded values. For a production setup, the document ID and secret would be retrieved from your server. This example also assumes that a JWT provider endpoint exists at '/jwt'. For information on setting up a JWT endpoint, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
 
 ```js
-const yourDocumentId = 'some-document-id';
-const yourSecretKey = 'your secret key';
 
 tinymce.init({
   selector: 'textarea',
-  plugin: 'rtc',
-  rtc_document_details_provider: () => Promise.resolve({ documentId: yourDocumentId }).
-  rtc_encryption_provider: ({documentId, sessionId}) => {
-    return Promise.resolve({ key: yourSecretKey });
-  },
-  rtc_token_provider: () => {
-    return fetch('/jwt', {
+  plugins: 'rtc',
+  rtc_document_id: 'unique-document-id',
+  rtc_encryption_provider: () => Promise.resolve({ key: 'a secret key' }),
+  rtc_token_provider: () =>
+    fetch('/jwt', {
       method: 'POST'
-    });
-  },
+    })
+    .then(response => response.json()),
```

Comment:
```suggestion
    .then((response) => response.json()),
```

Reply:
This was copy/paste from [the MDN example](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch). I didn't think we were applying our own code standards to examples for other developers 🤷‍♂️